### PR TITLE
Added indefinite rerolls and single explodes

### PR DESCRIPTION
**Pull request by scottlafetra**
See my PR for dice bag as well

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +76,7 @@ Dark Heresy 2nd edition syntaxes:
 Alias rolls are commands that are shorthand for a longer, more complex comand. They can also change what the dice faces appear as
 in most cases. Below is the complete list of aliases , with example rolls, currently supported by Dice Maiden. Have a game system that you want turned into an alias? Create an issue on github to get it added to this list!
 
-`4wod8` -> `4d10 f1 t8` World of darkness 4th edition. The first number is the number of dice to roll and the second is the toughness of the check. Exploding 10s will be added to this alias at a later date.
+`4wod8` -> `4d10 f1 t8 ie10` World of darkness 4th edition. The first number is the number of dice to roll and the second is the toughness of the check. Exploding 10s will be added to this alias at a later date.
```
Comment (Humblemonk) [inline README.md:92]: "Exploding 10s will be added to this alias at a later date." I am assuming this is no longer accurage?

Comment (scottlafetra) [reply README.md]: Whoops forgot to remove that. Added a commit below that fixes it completely

File: README.md [hunk 2]
```diff
@@ -28,12 +28,16 @@ Below are examples of the dice roll syntax.
 
 `!roll 3d6 e6` : Roll three six-sided dice and explode on sixes. Some game systems call this 'open eneded' dice. If the number rolled is greater than or equal to the value given for this option, the die is rolled again and added to the total. If no number is given for this option, it is assumed to be the same as the number of sides on the die. Thus, '3d6 e' is the same as '3d6 e6'.
 
+`!roll 3d6 e6` : Roll three six-sided dice and explode on sixes indefinitly. 
+
 `!roll 3d10 d1` : Roll three ten-sided dice and drop one die. The lowest value will be dropped first.  **NOTE:** These dice are dropped before any dice are kept with the following `k` command. Order of operations is : roll dice, drop dice, keep dice
 
 `!roll 3d10 k2` : Roll three ten-sided dice and keep two. The highest value rolled will be kept. 
 
 `!roll 4d6 r2` : Roll four six-sided dice and reroll any that are equal to or less than two.
```
Comment (Humblemonk) [inline README.md:37]: Should this be updated now to say something like "reroll once any that are equal to or less than two"?

File: README.md [hunk 3]
```diff
@@ -28,12 +28,16 @@ Below are examples of the dice roll syntax.
 
 `!roll 3d6 e6` : Roll three six-sided dice and explode on sixes. Some game systems call this 'open eneded' dice. If the number rolled is greater than or equal to the value given for this option, the die is rolled again and added to the total. If no number is given for this option, it is assumed to be the same as the number of sides on the die. Thus, '3d6 e' is the same as '3d6 e6'.
 
+`!roll 3d6 e6` : Roll three six-sided dice and explode on sixes indefinitly.
```
Comment (Humblemonk) [inline README.md:31]: shouldnt this syntax be `!roll 3d6 ie6` ?

File: README.md [hunk 4]
```diff
@@ -28,12 +28,16 @@ Below are examples of the dice roll syntax.
 
 `!roll 3d6 e6` : Roll three six-sided dice and explode on sixes. Some game systems call this 'open eneded' dice. If the number rolled is greater than or equal to the value given for this option, the die is rolled again and added to the total. If no number is given for this option, it is assumed to be the same as the number of sides on the die. Thus, '3d6 e' is the same as '3d6 e6'.
```
Comment (Humblemonk) [inline README.md:29]: Just noticed this as well. Should the verbiage be updated here that explodes will only happen once now with this command?

File: README.md (hunk 3 above)
Comment (B-Rex) [inline README.md:31]:
```suggestion
`!roll 3d6 e6` : Roll three six-sided dice and explode on sixes indefinitely. 
```

**Comment by scottlafetra** [comment]
Ok should be good to merge now.

**Pull request merged**